api/tasks.go

```diff
@@ -1167,3 +1169,9 @@ type WorkloadIdentity struct {
 	ServiceName string        `hcl:"service_name,optional"`
 	TTL         time.Duration `mapstructure:"ttl" hcl:"ttl,optional"`
 }
+
+type Action struct {
+	Name    string   `hcl:"name,label"`
+	Command *string  `mapstructure:"command" hcl:"command"`
```

Comment:
The only reason we use pointers to primitive fields in the API package is to accommodate optional fields. But `Command` is a required field.

```suggestion
	Command string  `mapstructure:"command" hcl:"command"`
```